source/_integrations/simplefin.markdown

```diff
@@ -30,5 +30,7 @@ Within the SimpleFIN Interface, you can create a **Claim Token**, a one-time tok
 
 Each `account` will be set up as a device in Home Assistant, and it will contain the following sensors:
 
-- `balance` 
-
+|Sensor|Description|
+|-------|---------------|
+|Balance|Account Balance|
+|Age| This sensor shows when the data was retrieved by the SimpleFin API |
```

Comment:
age: does this say something like 2 days old? Or does it give a timestamp?

Reply:
![](https://private-user-images.githubusercontent.com/6491743/351798803-a286f356-8eff-4daa-991f-d87484713f7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0MzQyMDgsIm5iZiI6MTcyMjQzMzkwOCwicGF0aCI6Ii82NDkxNzQzLzM1MTc5ODgwMy1hMjg2ZjM1Ni04ZWZmLTRkYWEtOTkxZi1kODc0ODQ3MTNmN2QucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDczMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MzFUMTM1MTQ4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWZhMjY1YWRmNzRjYTFhNzk5OWY4YTAyMWQ1NGJkZTcwMGViY2U2MjdjNjhhZDk3YWExN2QwY2E2NmJiYWU1NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.uvz-TC2WLN9Es7nfB_Shx4dUxP6AvhXgP4EBldYfwb4)

Here is what it looks like